flag_engine/features/schemas.py

```diff
@@ -35,7 +37,7 @@ class Meta:
 
 
 class BaseFeatureStateSchema(Schema):
-    id = fields.Int()
+    featurestate_uuid = fields.Str(dump_default=uuid.uuid4)
```

Comment:
Do we need the `dump_default` here as well as the `default_factory` on the model field?

Reply:
Yeah, if we are dumping the whole django environment.

Reply:
Yep, ok, I think I see why we need both.